### PR TITLE
[hi] Add content/hi/docs/setup/production-environment/windows/_index.md

### PR DESCRIPTION
**Pull request by anubha-v-ardhan**
This PR is part of ongoing hi localization effort here #29353 

/language hi

### CODE REVIEW

File: content/hi/docs/setup/production-environment/windows/_index.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+title: "कुबेरनेट्स में विंडोज़"
```
Comment (mittalyashu) [inline content/hi/docs/setup/production-environment/windows/_index.md:2]:
```suggestion
title: कुबेरनेट्स में Windows
```

Windows is a trademark of Microsoft _(i think)_.

**Review by mittalyashu — CHANGES_REQUESTED** [review]

File: content/hi/docs/setup/production-environment/windows/_index.md (hunk 1 above)
Comment (anubha-v-ardhan) [reply content/hi/docs/setup/production-environment/windows/_index.md:2]: Oh yeah! sorry, I didn't thought about it

**Comment by mittalyashu** [comment]
LGTM

**Review by mittalyashu — APPROVED** [review]

**Comment by sftim** [comment]
I'm going to treat https://github.com/kubernetes/website/pull/29413#issuecomment-900497774 as an actual LGTM
/lgtm

**Comment by anubha-v-ardhan** [comment]
@sftim The commits were not squashed. Is it fine to approve without squash.

**Comment by sftim** [comment]
> The commits were not squashed into one. Is it ok to approve without squash.

2 commits? Fine to merge without a squash.

24 commits? I'd squash them.

In between: use judgement. The more meaningful those commit messages, the less I'm inclined to ask for a squash.

**Comment by anubha-v-ardhan** [comment]
Got it. Thank you

**Pull request merged**